packages/ktfmt/package.yaml

```diff
@@ -10,7 +10,7 @@ categories:
   - Formatter
 
 source:
-  id: pkg:github/facebook/ktfmt@v0.49
+  id: pkg:github/facebook/ktfmt@v0.50
```

Comment:
`0.50` since ktfmt did not bump the jar file version for 0.51 (see [here](https://github.com/facebook/ktfmt/releases/tag/v0.51))